packages/bpk-docs/src/components/DocsPageBuilder/DocsPageBuilder.js

```diff
@@ -62,18 +62,11 @@ const markdownToHTML = (readmeString, headerPrefix = '') =>
     { renderer, headerPrefix },
   );
 
-const toSassdocLink = props => (
-  <SassdocLink
-    sassdocId={props.sassdocId}
-    category={props.category}
-    transparentBackground
-  />
-);
-
-toSassdocLink.propTypes = {
-  sassdocId: PropTypes.string.isRequired,
-  category: PropTypes.string.isRequired,
-};
+const toSassdocLink = createFromType('sass', { transparentBackground: true });
+const toIosdocLink = createFromType('ios', { transparentBackground: true });
+const toAndroiddocLink = createFromType('android', {
+  transparentBackground: true,
```

Comment:
It looks like `transparentBackground` is always true - maybe we should just remove that prop?